cirq/ops/controlled_gate.py

```diff
@@ -107,11 +150,19 @@ def _circuit_diagram_info_(self,
         if sub_info is None:
             return NotImplemented
         return protocols.CircuitDiagramInfo(
-            wire_symbols=('@',) + sub_info.wire_symbols,
+            wire_symbols=('@',)*self.total_control_qubits +
+                         sub_info.wire_symbols,
             exponent=sub_info.exponent)
 
     def __str__(self):
-        return 'C' + str(self.sub_gate)
+        return 'C'*self.total_control_qubits + str(self.sub_gate)
 
     def __repr__(self):
-        return 'cirq.ControlledGate(sub_gate={!r})'.format(self.sub_gate)
+        if (len(self.control_qubits) is 0 and
+            self.num_unspecified_control_qubits is 1):
+            return 'cirq.ControlledGate(sub_gate={!r})'.format(self.sub_gate)
+        else:
```

Comment:
nit: You can remove this else and reduce indentation since preceding block ends in a return.